deltachat-ios/AppDelegate.swift

```diff
@@ -413,23 +413,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
 
     // MARK: - Handle notification banners
 
-    private func userNotificationCenter(_: UNUserNotificationCenter, willPresent _: UNNotification, withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void) {
+    func userNotificationCenter(_: UNUserNotificationCenter, willPresent _: UNNotification, withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void) {
```

Comment:
is this callback about the "foreground notifications" that are working now? what are "foreground notifications" at all?

maybe add some lines of documentation about when this is called and why we call the completion handler in the given way.